kythe/extractors/bazel/Dockerfile

```diff
@@ -39,7 +39,7 @@ ADD kythe/extractors/bazel/bazel_wrapper.sh /kythe/
 ADD kythe/release/base/fix_permissions.sh /kythe/
 
 # Bazelisk
-ADD external/com_github_bazelbuild_bazelisk/linux_amd64_stripped/bazelisk /kythe/
+ADD external/com_github_bazelbuild_bazelisk/bazelisk_/bazelisk /kythe/
```

Comment:
Is this required in this CL or could/should this be a separate PR?

Reply:
I can split it out - bazel changed the path name, so this fix is needed for the image to build.

Reply:
https://github.com/kythe/kythe/pull/4555